dashboards-observability/public/components/application_analytics/home.tsx

```diff
@@ -0,0 +1,148 @@
+  availabilityMetrics: string;
+  dateCreated: string;
+  dateModified: string;
+};
+
+const dateString = new Date().toISOString();
+
+const dummyApplication: ApplicationType[] = [{
+  name: "Cool Application", 
+  id: "id", 
+  composition: "Payment, user_db",
+  currentAvailability: "Available",
+  availabilityMetrics: "Error rate: 0.80%, Throughput: 0.94%, Latency: 600ms",
+  dateCreated: dateString, 
+  dateModified: dateString
+}];
+
+export const Home = (props: HomeProps) => {
+  const { pplService, dslService, timestampUtils, savedObjects, parentBreadcrumb, http, chrome, notifications } = props;
+  const [indicesExist, setIndicesExist] = useState(true);
+  const storedFilters = sessionStorage.getItem('AppAnalyticsFilters');
+  const [query, setQuery] = useState<string>(sessionStorage.getItem('AppAnalyticsQuery') || '');
+  const [filters, setFilters] = useState<FilterType[]>(
+    storedFilters ? JSON.parse(storedFilters) : []
+  );
+  const [startTime, setStartTime] = useState<string>(
+    sessionStorage.getItem('AppAnalyticsStartTime') || 'now-24h'
+  );
+  const [endTime, setEndTime] = useState<string>(
+    sessionStorage.getItem('AppAnalyticsEndTime') || 'now'
+  );
+
+  const setFiltersWithStorage = (newFilters: FilterType[]) => {
+    setFilters(newFilters);
+    sessionStorage.setItem('AppAnalyticsFilters', JSON.stringify(newFilters));
+  };
+  const setQueryWithStorage = (newQuery: string) => {
+    setQuery(newQuery);
+    sessionStorage.setItem('AppAnalyticsQuery', newQuery);
+  };
+  const setStartTimeWithStorage = (newStartTime: string) => {
+    setStartTime(newStartTime);
+    sessionStorage.setItem('AppAnalyticsStartTime', newStartTime);
+  };
+  const setEndTimeWithStorage = (newEndTime: string) => {
+    setEndTime(newEndTime);
+    sessionStorage.setItem('AppAnalyticsEndTime', newEndTime);
+  };
+  
+  useEffect(() => {
+    handleIndicesExistRequest(http, setIndicesExist);
+  }, []);
+
+  const commonProps: AppAnalyticsComponentDeps = {
+    parentBreadcrumb: parentBreadcrumb,
+    http: http,
+    chrome: chrome,
+    query,
+    setQuery: setQueryWithStorage,
+    filters,
+    setFilters: setFiltersWithStorage,
+    startTime,
+    setStartTime: setStartTimeWithStorage,
+    endTime,
+    setEndTime: setEndTimeWithStorage,
+    indicesExist,
+  };
+
+  return (
+    <div>
+      <Switch>
+        <Route
+          exact
+          path={['/', '/application_analytics']}
+          render={() => 
+            <ObservabilitySideBar>
+            <AppTable loading={false} applications={dummyApplication} {...commonProps} />
+            </ObservabilitySideBar>
+          }
+        />
+        <Route
+          exact
+          path={'/application_analytics/create'}
+          render={() => 
+            <CreateApp
+            dslService={dslService}
+            {...commonProps}
+            />
+          }
+        />
+        <Route
+          exact
+          path={'/application_analytics/:id+'}
```

Comment:
Question: Just curious what's the plus after id for?

Reply:
If the id includes a "/", the + makes sure it is not seen as the end of the id and gets the full id including the "/" and everything after